rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -26,14 +37,28 @@
       });
     }
 
+    function pollForResponses() {
+      $.get(responsesUrl).then(function(newState) {
+        state = Object.assign({}, state, newState);
+        render();
+        if (state.is_open) {
+          setTimeout(pollForResponses, 3000);
+        }
+      }).fail(function () {
+        // TODO: try again?
+        console.error("Error retrieving response data");
+      });
+    }
+
     $(function($) { // onLoad
       var block = $element.find(rapidTopLevelSel);
       var isOpen = block.attr('data-open') === 'True';
       var isStaff = block.attr('data-staff') === 'True';
-      render({
-        is_open: isOpen,
-        is_staff: isStaff
-      });
+      state.is_open = isOpen;
+      state.is_staff = isStaff;
```

Comment:
I would prefer that we skip creating the `isOpen` and `isStaff` vars here and just use `Object.assign` et al to set the values on `state` directly

Reply:
Sounds good